### DISCUSSION TITLE
Impossible to install NodeJS

### DISCUSSION THREAD

**Discussion by lad75020** [nodejs/help#4913] [Q&A]
### Version

17.7.1

### Platform

Microsoft Windows NT 10.0.22000.0 x64

### What steps will reproduce the bug?

Run the installer

### How often does it reproduce? Is there a required condition?

Always

### What is the expected behavior?

INsatllation of NodeJS

### What do you see instead?

INstallation starts normally, I select all options in the installer, including chocolatey. At what seems the final step of install, the program rolls back and suggests to retry, which I have done many times after rebooting etc..
<img width="386" alt="Capture d’écran 2022-03-16 080325" src="https://user-images.githubusercontent.com/5177159/158534733-97a8d3af-05cc-491d-b400-d038716c0fb5.png">
.

### Additional information

Where can I find the install logs?

**Comment by hello-smile6** [comment]
Download the Node.js `.zip` binary release and manually install Node.js to the `Program Files` directory. Then, run `install-tools.cmd`. That should work, assuming my memory is correct.
